docs/user-guide/etendo-classic/optional-features/bundles/spain-localization/overview.md

```diff
@@ -65,6 +65,7 @@ El bundle de Localización Española contiene los módulos que añaden funcional
 - OpenID Service Integration Translation Spanish Spain
 - Integration with Google APIs Translation Spanish Spain
 - Openbravo 3.0 Translation Pack: Spanish Spain (es_ES)
+- TicketBai
```

Comment:
This could also be a link to the page and move it along with the pages that have documentation above.